File: Iniciante/1060.c
#include <stdio.h>
 
int main() {
    
    int cont=0, i;
    float n;

    for(i = 0; i < 6; i++){
        scanf("%f", &n);
        if(n > 0) cont++;
    }

    printf("%d valores positivos\n", cont);

    return 0;
}

File: Iniciante/1060.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        int cont=0;
        
        for(int i = 0; i < 6; i++){
            float n = scanf.nextFloat();
            if(n > 0) cont++;
        }

        System.out.printf("%d valores positivos\n", cont);

    }
}